### ISSUE TITLE
Cannot destructure property 'translate' of 'this.draggedElm' as it is undefined

### ISSUE THREAD

**Issue by ddenev** [dflex-js/dflex#729]
I have the following Vue component:
```vue
<template>
  <component :is="tag" style="touch-action: none" @pointerdown="onPointerDown">
    <template v-for="slot of Object.keys($slots)" #[slot]="scope">
      <slot :name="slot" v-bind="scope || {}" />
    </template>
  </component>
</template>

<script setup>
import { onUnmounted } from 'vue'
import { store, DnD } from '@dflex/dnd'

const props = defineProps({
  tag: { type: [String, Object], required: true },
  id: { type: String, required: true }
})

store.register({ id: props.id })
onUnmounted(() => {
  store.unregister(props.id)
})
let dndSession = null

function onPointerDown (event) {
  event.currentTarget.setPointerCapture(event.pointerId)
  event.currentTarget.addEventListener('pointermove', onPointerMove)
  event.currentTarget.addEventListener('pointerup', onPointerUp)
  dndSession = new DnD(props.id, { x: event.clientX, y: event.clientY })
}
function onPointerMove (event) {
  // dndSession.dragAt(event.clientX, event.clientY)
}
function onPointerUp (event) {
  event.currentTarget.removeEventListener('pointermove', onPointerMove)
  event.currentTarget.removeEventListener('pointerup', onPointerUp)
  dndSession.endDragging()
  dndSession = null
}
</script>
```

When I click on the element (`pointerdown`) I get the following error:
![image](https://github.com/dflex-js/dflex/assets/1393766/4ca97ffd-7ea7-4591-95a7-71e08b508904)

Am I doing something wrong?

**Comment by jalal246** [comment]
Hi @ddenev, you are using it in production mode that's why error messages are not clear. I will try to check it out and get back to you.

**Comment by jalal246** [comment]
It appears that a generated child text node is triggering an error during registration. Given my limited familiarity with Vue, it seems this node is automatically generated by the template.

For instance, this list triggers an error due to the presence of text elements:

```js
NodeList(6) [text, li#item1, li#item2, li#item3, li#item4, text]
```
The inclusion of non-HTML Element nodes, such as text nodes, can disrupt the expected behavior and potentially lead to errors in the process.

DFlex won't allow it assuming all elements in the same container should be HTML Element.

https://github.com/dflex-js/dflex/blob/927cab38a0580d15b49b4f177e724e0f70c0aa19/packages/dflex-store/src/DFlexBaseStore.ts#L231C1-L259C1

**Comment by ddenev** [comment]
But there are no text elements in the rendered DOM:
![image](https://github.com/dflex-js/dflex/assets/1393766/1cc0b08a-fc64-491c-b335-07b2a8b08ce4)

**Comment by jalal246** [comment]
I know, but if you point to the parent element and then do `$0.childNodes` you can see there's one. It's an easy fix assuming this is a Vue default behavior and not something that can be addressed within the template itself.

**Comment by ddenev** [comment]
If it's an easy fix, could you please do it then? There is nothing I can do at my end since indeed, this is how Vue works: https://github.com/vuejs/core/issues/8444

**Comment by jalal246** [comment]
Thanks for sharing the link. I will ship and release it to npm tomorrow.

**Comment by ddenev** [comment]
Thank you, @jalal246!

**Comment by ddenev** [comment]
Hi @jalal246,

just updated to v3.9.10 but still have the same error.

**Comment by jalal246** [comment]
Are you sure? I made a quick sandbox and it's working: https://codesandbox.io/s/loving-almeida-sst43z?file=/src/components/ListItem.vue

**Comment by ddenev** [comment]
Thank you for the sandbox sample! I noticed that you have added an `id` prop to the `component` element
![image](https://github.com/dflex-js/dflex/assets/1393766/da29e07c-44e0-41c5-a94f-c931ba7b4e0a)
which I did not have in my template.

Adding it to my template removed the error. Without it I still get the same error.

I never met this requirement in the documentation though - that the draggable element should have an `id` attribute - or maybe I wasn't paying enough attention :)
